### DISCUSSION TITLE
Battery Charge on mac

### DISCUSSION THREAD

**Discussion by StoneMan240** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1281] [General]
Hello,

So basically, if I set the charge limit to 80% for Al Denate, the charging will stop at 80%. However, if I update macOS or shut down my Mac, will it resume charging?

**Comment by MatthiasKerbl** [comment]
Hi @StoneMan240,

If you have an Apple Silicon MacBook, this is correct.

Best,
Matthias

**Reply by StoneMan240** [reply]
I’m on apple silicon mac
